infrastructure/code/convert.go

```diff
@@ -412,20 +413,21 @@ func (s *SarifConverter) getIgnoreDetails(result codeClientSarif.Result) (bool,
 		if suppression.Properties.Expiration != nil {
 			expiration = *suppression.Properties.Expiration
 		}
-		ignoredOn, err := time.Parse(time.RFC3339, suppression.Properties.IgnoredOn)
+		ignoredOn, err := time.Parse("Mon Jan 02 2006", suppression.Properties.IgnoredOn)
```

Comment:
This change is because the date is coming as a string in this format: `"Wed Jun 05 2024"`